ref/Boolean_Operators.md

```diff
@@ -181,7 +181,14 @@ This operator is similar to **&gt;** but tests for **less than or equal to**.
 
 ------
 
-#####  Fuzzy comparisons: `~=`, `~!=`, `~<`, `~>`, `~>=`, `~<=`
+####  Fuzzy comparisons:
+
+##### Approximately equal: `‹expr1› ~= ‹expr2›`
+##### Not approximately equal: `‹expr1› ~!= ‹expr2›`
```

Comment:
That was a deliberate modification. To me, “approximately equal” is a common term which has a well-defined meaning: equal up to some fuzz. “Approximately not equal”, on the other hand, is a rather unusual term, so one has to start thinking. Is it just *approximately* unequal, but not *really exactly* unequal? How so? Looks unequal at first glance but turns out to be equal on closer inspection? Yes, you could rule out such interpretations pretty quickly, but the fact that you could start thinking along these lines indicates potential confusion. On the other hand, negating an expression is common enough, so negating “approximately equal” we obtain “not approximately equal”.

@kranich: The same for the other approximate comparisons: I rephrased them to be combinations of an exact comparison and approximate equality. For the strict fuzzy comparisons (strange combination of words!), I realize that the placement of the negation allows for a second alternative: instead of my “not greater or approximately equal” one could write “less and not approximately equal”, if you think that's an improvement.

Your statement about the order of symbols is of course quite valid; I hadn't considered that aspect. I find it difficult to judge between these goals. If you uphold your request even after hearing about my motivation, and noone else agrees with me soon enough, I'll concede to that.

By the way, what documentation are you referring to? The printed book? I thought that was generated from the web site, as were our documentation files. And I couldn't find any occurrence of “approximately” in our files. So where does the documentation have a spelled out name for these operators?